docs/quick_start.md

```diff
@@ -51,23 +51,13 @@ export OPEN_AEA_IPFS_ADDR="/dns/registry.autonolas.tech/tcp/443/https"
 touch Pipfile && pipenv --python 3.10 && pipenv shell
 ```
 
-3. Install {{open_autonomy}}.
+3. Install {{open_autonomy}}
 ```bash
 pip install open-autonomy
 ```
 
-4. Initialize the remote IPFS registry.
-```bash
-autonomy init --remote
-```
-
 ## Deploy a Local Agent Service
 
-!!! note
-    On **MacOS** and **Windows**, running Docker containers requires having Docker Desktop running as well. If you're using one of those systems, remember to start Docker Desktop
-    before you run agent services.
-
-
```

Comment:
Why are we removing this from the quickstart?

Reply:
Looks like the merge errors, I'll revert them